Criipto.Signatures/Operations.cs

```diff
@@ -120,31 +223,133 @@ public class CloseSignatureOrderMutation {
         mutation closeSignatureOrder($input: CloseSignatureOrderInput!) {
           closeSignatureOrder(input: $input) {
             signatureOrder {
+              ...BasicSignatureOrder
+              documents {
+                ...BasicDocument
+                ...SignedDocument
+              }
+            }
+          }
+        }
+        fragment BasicDocument on Document {
+          __typename
+          id
+        }
+        fragment SignedDocument on Document {
+          blob
+          signatures {
+            __typename
+            signatory {
               id
-              signatories {
+            }
+            ... on JWTSignature {
+              jwt
+              jwks
+            }
+          }
+        }
+        fragment BasicSignatory on Signatory {
+          id
+          status
+          href
+          downloadHref
+          reference
+          role
+          evidenceProviders {
+            __typename
+            id
+          }
+          documents {
+            edges {
+              status
+              node {
+                __typename
                 id
-                status
-                downloadHref
               }
+            }
+          }
+        }
+        fragment BasicSignatureOrder on SignatureOrder {
```

Comment:
Is the `Basic` prefix a naming convention typically used in GQL ?

Reply:
@sgryt No, just the best name I could come up with for `Signature order without document blobs`

Reply:
How about `EmptySignatureOrder` ?

Reply:
It's not quite empty though, there's plenty of data there @sgryt
It's not something the user of the SDK will see, it was mostly to reduce the amount of duplicate GraphQL code and make sure return values are consistent.

Reply:
OK, I wasn't sure about the visibility for consumers. Disregard the suggestion, in that case.